setup.py

```diff
@@ -12,12 +12,36 @@
 if version is None:
     raise RuntimeError('Could not determine version')
 
+dependencies = [
+    'numpy',
+    'scipy',
+    'matplotlib',
+    'pandas',
+    'pyopengl',
+    'pyglet~=1.3.0',
+    'pillow',
+    'moviepy',
+    'lxml',
+    'openpyxl',
+    'configobj',
+    'psychopy==3.0.0',
+    'pytest',
+    'pytest-cov',
+]
+
+
 setup(name='sp_experiment',
       version=version,
       description='Implemetation of the Sampling Paradigm in PsychoPy',
       url='http://github.com/sappelhoff/sp_experiment',
       author='Stefan Appelhoff',
       author_email='stefan.appelhoff@mailbox.org',
       license='BSD 3-Clause License',
+      classifiers=[
+          'Programming Language :: Python :: 3.6',
+          'Operating System :: Microsoft :: Windows',
+      ],
       packages=find_packages(),
+      install_requires=dependencies,
+      package_data={'': ['*.json']},
```

Comment:
About that: I thought that I should rewrite this JSON file to be a python file that has a function to output a JSON file (with the same contents), see #1

Reply:
I didn't dive into the code and just tried to make things build. :)

Is the data from this json file supposed to be configurable by users or is it just static for all experiments?

Reply:
It's a static: A dictionary that contains verbose description about each column that is saved in the data log files